src/Bridge/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -598,7 +598,7 @@ private function addDefaultsSection(ArrayNodeDefinition $rootNode): void
                 return $normalizedDefaults;
             });
 
-        foreach (ApiResource::CONFIGURABLE_DEFAULTS as $attribute) {
+        foreach (ApiResource::getConfigMetadata()[1] as $attribute => $_) {
```

Comment:
[1] isn't so clear, I wish we could [, $configurableAttributes]; $configurableAttributes as ... :/

Reply:
We don't have something better for now.